src/harmony/Checkbox/Checkbox.tsx

```diff
@@ -12,17 +13,16 @@ interface CheckboxProps extends React.InputHTMLAttributes<HTMLInputElement> {
 
 export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
     ({ label, className, view = 'default', ...rest }, ref) => (
-        <label
+        <Badge
```

Comment:
Why we need to use Badge everywhere?

Reply:
so that Badge, Checkbox, Radio looks the same